src/client.ts

```diff
@@ -4097,4 +4102,64 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
 
     return await this.post<QueryDraftsResponse<StreamChatGenerics>>(this.baseURL + '/drafts/query', payload);
   }
+
+  /**
+   * createReminder - Creates a reminder for a message
+   *
+   * @param {string} messageId The ID of the message to create reminder for
+   * @param {CreateReminderOptions} options The options for creating the reminder
+   * @returns {Promise<ReminderResponse>}
+   */
+  async createReminder<StreamChatGenerics extends ExtendableGenerics = DefaultGenerics>(
+    messageId: string,
+    options: CreateReminderOptions = {},
+  ): Promise<ReminderResponse<StreamChatGenerics>> {
+    return await this.post<ReminderResponse<StreamChatGenerics>>(
+      `${this.baseURL}/messages/${messageId}/reminders`,
+      options,
+    );
+  }
+
+  /**
+   * updateReminder - Updates an existing reminder for a message
+   *
+   * @param {string} messageId The ID of the message whose reminder to update
+   * @param {UpdateReminderOptions} options The options for updating the reminder
+   * @returns {Promise<ReminderResponse>}
+   */
+  async updateReminder<StreamChatGenerics extends ExtendableGenerics = DefaultGenerics>(
```

Comment:
@nijeesh-stream please remove the generics as they are not supported by the SDK anymore. Thank you